doc/lightning-listchannels.7.txt

```diff
@@ -0,0 +1,65 @@
+LIGHTNING-LISTCHANNELS(7)
+=======================
+:doctype: manpage
+
+NAME
+----
+lightning-listchannels - Command for returning data on active lightning
+channels..
+
+SYNOPSIS
+--------
+*listchannels* ['short_channel_id']
+
+DESCRIPTION
+-----------
+The *listchannels* RPC command returns data on channels that are known to the
+node.
+
+If no 'short_channel_id' is supplied, then data on all lightning channels that
+are known to this node, are returned.
+
+Supplying 'short_channel_id' will filter the results to only return data on a
+channel with a matching 'short_channel_id', if one exists.
```

Comment:
We could add a mention that channel directions are considered independently, hence even specifying a `short_channel_id` might results in 0, 1 or 2 resulting channel directions being returned.